### ISSUE TITLE
[MDBottomNavigation] [Android] onViewAttachedToWindow error onPause

### ISSUE THREAD

**Issue by chrisalexqm** [nativescript-community/ui-material-components#262]
### Which platform(s) does your issue occur on?
- Android device and emulator

**Error:**

```
System.err: An uncaught Exception occurred on "main" thread.
System.err: Calling js method onViewAttachedToWindow failed
System.err: Error: java.lang.IllegalArgumentException: No view found for id 0x2 (unknown) for fragment FragmentBase_vendor_64_26_TabFragmentImplementation{8c9bbe0} (9f0ec83b-60f7-4f9b-868d-4e500014cd3d) id=0x2 android:bottomnavigation:2:0}
System.err:
System.err: StackTrace:
System.err: setPrimaryItem(file: node_modules\@nativescript-community\ui-material-bottom-navigation\index.android.js:420:25)
System.err:     at changeTab(file: node_modules\@nativescript-community\ui-material-bottom-navigation\index.android.js:392:13)
System.err:     at _onAttachedToWindow(file: node_modules\@nativescript-community\ui-material-bottom-navigation\index.android.js:308:13)
System.err:     at AttachListener.onViewAttachedToWindow(file: node_modules\@nativescript-community\ui-material-bottom-navigation\index.android.js:157:18)
System.err:     at com.tns.Runtime.callJSMethodNative(Native Method)
System.err:     at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1302)
System.err:     at com.tns.Runtime.callJSMethodImpl(Runtime.java:1188)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1175)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1153)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1149)
System.err:     at com.tns.gen.java.lang.Object_vendor_216_26_AttachListener.onViewAttachedToWindow(Object_vendor_216_26_AttachListener.java:18)
System.err:     at android.view.View.dispatchAttachedToWindow(View.java:21318)
System.err:     at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:4239)
System.err:     at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:4246)
System.err:     at android.view.ViewGroup.addViewInner(ViewGroup.java:6001)
System.err:     at android.view.ViewGroup.addView(ViewGroup.java:5777)
System.err:     at android.view.ViewGroup.addView(ViewGroup.java:5717)
System.err:     at android.view.ViewGroup.addView(ViewGroup.java:5690)
System.err:     at android.view.ViewOverlay$OverlayViewGroup.add(ViewOverlay.java:208)
System.err:     at android.view.ViewGroupOverlay.add(ViewGroupOverlay.java:72)
System.err:     at androidx.transition.ViewGroupOverlayApi18.add(ViewGroupOverlayApi18.java:48)
System.err:     at androidx.transition.Visibility.onDisappear(Visibility.java:428)
System.err:     at androidx.transition.Visibility.createAnimator(Visibility.java:257)
System.err:     at androidx.transition.Transition.createAnimators(Transition.java:747)
System.err:     at androidx.transition.TransitionSet.createAnimators(TransitionSet.java:480)
System.err:     at androidx.transition.Transition.playTransition(Transition.java:1821)
System.err:     at androidx.transition.TransitionManager$MultiListener.onPreDraw(TransitionManager.java:301)
System.err:     at android.view.ViewTreeObserver.dispatchOnPreDraw(ViewTreeObserver.java:1102)
System.err:     at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:3310)
System.err:     at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:2200)
System.err:     at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:9065)
System.err:     at android.view.Choreographer$CallbackRecord.run(Choreographer.java:999)
System.err:     at android.view.Choreographer.doCallbacks(Choreographer.java:797)
System.err:     at android.view.Choreographer.doFrame(Choreographer.java:732)
System.err:     at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:984)
System.err:     at android.os.Handler.handleCallback(Handler.java:883)
System.err:     at android.os.Handler.dispatchMessage(Handler.java:100)
System.err:     at android.os.Looper.loop(Looper.java:237)
System.err:     at android.app.ActivityThread.main(ActivityThread.java:8016)
System.err:     at java.lang.reflect.Method.invoke(Native Method)
System.err:     at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:496)
System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1087)
System.err: Caused by: java.lang.IllegalArgumentException: No view found for id 0x2 (unknown) for fragment FragmentBase_vendor_64_26_TabFragmentImplementation{8c9bbe0} (9f0ec83b-60f7-4f9b-868d-4e500014cd3d) id=0x2 android:bottomnavigation:2:0}
System.err:     at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:315)
System.err:     at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1199)
System.err:     at androidx.fragment.app.FragmentManager.performPendingDeferredStart(FragmentManager.java:1123)
System.err:     at androidx.fragment.app.Fragment.setUserVisibleHint(Fragment.java:1206)
System.err:     ... 38 more
````

### Please, provide the following version numbers that your issue occurs with:

NS-Vue 2.9.0 / NS 8.0.1

### Please, tell us how to recreate the issue in as much detail as possible. 
Add "@nativescript-community/ui-material-bottom-navigation": "^5.2.14" to your vue app and use it like this:

```
<Page>
  <ActionBar title="Tesssst"></ActionBar>
  <MDBottomNavigation
    selectedIndex="0"
    selectedIndexChanged="onSelectedIndexChanged"
  >
    <!-- The bottom tab UI is created via TabStrip (the containier) and TabStripItem (for each tab)-->
    <MDTabStrip>
      <MDTabStripItem>
        <Label text="Home"></Label>
      </MDTabStripItem>
      <MDTabStripItem>
        <Label text="Account"></Label>
      </MDTabStripItem>
      <MDTabStripItem>
        <Label text="Search"></Label>
      </MDTabStripItem>
    </MDTabStrip>

    <!-- The number of TabContentItem components should corespond to the number of TabStripItem components -->
    <MDTabContentItem>
      <Frame>
        <Page>
          <GridLayout>
            <Label text="Home Pages" class="h2 text-center"></Label>
          </GridLayout>
        </Page>
      </Frame>
    </MDTabContentItem>
    <MDTabContentItem>
      <Frame>
        <Page>
          <GridLayout>
            <Label text="Account Page" class="h2 text-center"></Label>
          </GridLayout>
        </Page>
      </Frame>
    </MDTabContentItem>
    <MDTabContentItem>
      <Frame>
        <Page>
          <GridLayout>
            <Label text="Search Page" class="h2 text-center"></Label>
          </GridLayout>
        </Page>
      </Frame>
    </MDTabContentItem>
  </MDBottomNavigation>
</Page>
```

Then... change to other app or simply minimize the NS app and this error will appear.

**Note:**
I saw there is an other similar issue with Tabs, the only thing I understand about this error is that it is related to fragments management.

**PDT:** I tried to use this MDBottomNavigation without a `<Frame>` wrapper, with a `<Frame>` wrapper and a `<Page>` wrapper. In those 3 cases I mentioned, the error keeps getting. Only removing the MDBottomNavigation this error dissapear.

**Comment by adeyemiadetayoccl** [comment]
So, I had to battle this issue for some hours today, after doing an upgrade to NS 8, my fix is, I brought back the BottomNavigation (and tab-navigation-base, bottom-navigation depends on it) into nativescript-core (both were removed in ns 8)

Some minor changes will have to be made to the code , I have attached working versions of the two folders , just extract the zips and drop the two folders in  "...\node_modules\@nativescript\core\ui" then you can continue with BottomNavigation (just as is from ns 7)

[bottom-navigation.zip](https://github.com/nativescript-community/ui-material-components/files/6254895/bottom-navigation.zip)
[tab-navigation-base.zip](https://github.com/nativescript-community/ui-material-components/files/6254896/tab-navigation-base.zip)

**Comment by chrisalexqm** [comment]
> So, I had to resolve this issue for some hours today, after doing an upgrade to NS 8, my fix is, I brought back the BottomNavigation (and tab-navigation-base, bottom-navigation depends on it) into nativescript-core (both were removed in ns 8)
> 
> Some minor changes will have to be made to the code , I have attached working versions of the two folders , just extract the zips and drop the two folders in "...\node_modules@nativescript\core\ui" then you can continue with BottomNavigation (just as is from ns 7)
> 
> [bottom-navigation.zip](https://github.com/nativescript-community/ui-material-components/files/6254895/bottom-navigation.zip)
> [tab-navigation-base.zip](https://github.com/nativescript-community/ui-material-components/files/6254896/tab-navigation-base.zip)

Nice, thanks for this workaround. I will test it.

**Comment by lehongwang** [comment]
> So, I had to resolve this issue for some hours today, after doing an upgrade to NS 8, my fix is, I brought back the BottomNavigation (and tab-navigation-base, bottom-navigation depends on it) into nativescript-core (both were removed in ns 8)
> 
> Some minor changes will have to be made to the code , I have attached working versions of the two folders , just extract the zips and drop the two folders in "...\node_modules@nativescript\core\ui" then you can continue with BottomNavigation (just as is from ns 7)
> 
> [bottom-navigation.zip](https://github.com/nativescript-community/ui-material-components/files/6254895/bottom-navigation.zip)
> [tab-navigation-base.zip](https://github.com/nativescript-community/ui-material-components/files/6254896/tab-navigation-base.zip)
there are alot of changes, bring back two those folders could make bottom-navigation work. :((

**Comment by vallemar** [comment]
For me this error is corrected with version v5.2.16  🚀

**Comment by chrisalexqm** [comment]
> For me this error is corrected with version v5.2.16 🚀

I just tried v5.2.16 and this error is solved 🎉.

PD: There are logs in the last commit [4c8932b](https://github.com/nativescript-community/ui-material-components/commit/4c8932b25fa97dd97a08ac9b4b828e0948d2241e).

**Comment by farfromrefug** [comment]
@chrisalexqm logs should be fixed in 5.2.17
